app/status/status.go

```diff
@@ -0,0 +1,107 @@
+// Cosi takes a file or a message and signs it collectively.
+// For usage, see README.md
+package main
+
+import (
+	"os"
+
+	"errors"
+	"time"
+
+	"github.com/dedis/cothority/app/lib/config"
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/sda"
+
+	"fmt"
+
+	"github.com/dedis/cothority/services/status"
+	"gopkg.in/codegangsta/cli.v1"
+)
+
+// RequestTimeOut defines when the client stops waiting for the CoSi group to
+// reply
+const RequestTimeOut = time.Second * 10
+
+const optionGroup = "group"
+const optionGroupShort = "g"
+
+func init() {
+	log.SetDebugVisible(1)
+	log.SetUseColors(false)
+}
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "Status"
+	app.Usage = "Get and print status of all servers in a file."
+	//a status is a list of connections and packets sent and received for each server in the file
+	app.Commands = []cli.Command{
+		{
+			Name:    "network",
+			Aliases: []string{"g"},
+			Usage:   "Gets status from all entities in group file.",
+			Action:  network,
+		},
+	}
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  optionGroup + " ," + optionGroupShort,
+			Value: "group.toml",
+			Usage: "Cothority group definition in `FILE.toml`",
+		},
+		cli.IntFlag{
+			Name:  "debug, d",
+			Value: 0,
+			Usage: "debug-level: `integer`: 1 for terse, 5 for maximal",
+		},
+	}
+	app.Before = func(c *cli.Context) error {
+		log.SetDebugVisible(c.GlobalInt("debug"))
+		return nil
+	}
+	app.Action = network
+	app.Run(os.Args)
+}
+
+// signFile will search for the file and sign it
+// it always returns nil as an error
+func network(c *cli.Context) error {
```

Comment:
the first word in the comment should be the name of the method. Punctuation and lower/uppercase are very welcome. And you should write it so that even Swiss/German/French people can understand it:

```
// network will contact all cothorities in the group-file and print
// the status-report of each one.
```